vm-setup/roles/v1aX_integration_test/templates/controlplane_ubuntu.yaml

```diff
@@ -95,6 +95,8 @@ spec:
               ethernets:
                   enp2s0:
                       dhcp4: true
+                  enp1s0:
+                      dhcp4: false
```

Comment:
I would rather add it in the 60, so that it is with the bridge. 50- comes from cloud-init normally, and might be overwritten

Reply:
Sorry, I did not get you, what do you mean by  50- comes from cloud-init normally, and might be overwritten? I think the naming is just to tell which file to be applied before. In our case, 50 would be applied before 60. I don't think 50- would be overwritten automatically, unless we do some changes explicitly on the interface in cloud-init

Reply:
I meant that the 50-cloud-init.yaml file is normally generated by cloud-init.

Reply:
so creating another file called the same way should be done only if you want to overwrite the cloud-init output. And there might be unexpected cases where cloud-init could regenerate that file and overwrite your changes

Reply:
That holds true for the whole file then, not this change I suppose

Reply:
I have changed the file names @maelk

Reply:
how does the numbering apply ? because we want this configuration to have a higher priority than the 50-cloud-init.yaml one

Reply:
I just found these in netplan 
"Configuration is a merge of all /etc/netplan/\*.yaml. The netplan config show command will read the current on-disk configuration and merge any configs in /etc/netplan/\*.yaml merge them and produce a YAML formatted output to stdout."

Reply:
netplan’s configuration files use the YAML format. All /{lib,etc,run}/netplan/*.yaml are considered. Lexicographically later files (regardless of in which directory they are) amend (new mapping keys) or override (same mapping keys) previous ones. A file in /run/netplan completely shadows a file with same name in /etc/netplan, and a file in either of those directories shadows a file with the same name in /lib/netplan.

Reply:
changed it to 51 and 52 just to be on the safe side

Reply:
good. Maybe we could later on actually merge those two.